test_regress/driver.pl

```diff
@@ -28,6 +28,11 @@ BEGIN
 
 eval "use Parallel::Forker; \$Fork=Parallel::Forker->new(use_sig_child=>1); \$::Have_Forker=1;";
 $Fork = Forker->new(use_sig_child=>1) if !$Fork;
+my $forker_Min_Version = 1.258;
+if ($::Have_Forker && $Parallel::Forker::VERSION < $forker_Min_Version) {
+    print STDERR "Parallel::Forker version is too old () requires $forker_Min_Version or newer\n";
```

Comment:
Thanks for fixing this, it only breaks the code coverage but might as well encourage proper upgrades.

I'd suggest we recommend how to fix it.

```suggestion
    print STDERR "driver.pl: Parallel::Forker is older than $forker_Min_Version, suggest 'cpan install Parallel::Forker'\n";
```

If this works/passes feel free to push.